### ISSUE TITLE
[FEATURE] Add custom video to landing page for app overview

### ISSUE THREAD

**Issue by manuelalferez** [manuelalferez/flowinance#58]
### Is this a unique feature?

- [X] I have checked "open" AND "closed" issues and this is not a duplicate

### Is your feature request related to a problem/unavailable functionality? Please describe.

Currently, when you visit flowinance.com, users have to scroll down to clearly understand the platform's purpose and features.

### Proposed Solution

It would be more intuitive if we included a custom video player (not a YouTube embed) with a personalized video that showcases what users can do with the application. This would provide a clearer, more engaging introduction right away.

### Screenshots

<img width="1201" alt="image" src="https://github.com/user-attachments/assets/531a7a64-406b-4472-bef4-ce72186368de">

### Do you want to work on this issue?

No

**Comment by aten2005** [comment]
Do you have a video for the landing page?

**Comment by manuelalferez** [comment]
@aten2005 no, it must be created. If you know any platform that can do it easily..

**Comment by manuelalferez** [comment]
Before you get started, please take a moment to read the contributing guidelines: [Contributing Guide](https://github.com/manuelalferez/flowinance/blob/main/CONTRIBUTING.md).

**Comment by Anushlinux** [comment]
I can work on this pls assign this to me

**Comment by manuelalferez** [comment]
@Anushlinux I just assigned it to you. Let me know if I can assist you with anything!

**Comment by mahendra785** [comment]
i would like to work on this issue . can you please assign me on this

**Comment by manuelalferez** [comment]
Sure! Thanks @mahendra785

**Comment by manuelalferez** [comment]
Let me know if you need any assistance

**Comment by manuelalferez** [comment]
Here you have an example: 
<img width="1920" alt="image" src="https://github.com/user-attachments/assets/d265ca97-888b-444f-84ae-9ce357ee6a6c">

**Comment by mahendra785** [comment]
ohkk ill get on it
